spec/acceptance/00_backend_spec.rb

```diff
@@ -5,7 +5,8 @@
   context 'default' do
     it 'should work without errors' do
       pp = <<-EOS
-      include ::sensu::backend
+      class { '::sensu': }
+      class { '::sensu::backend': }
```

Comment:
curious why the change to resource syntax for the inclusion of the class.

Reply:
I wanted to verify the README style of examples would work.  If `class {}` works then we know `include` will work.